pavelib/prereqs.py

```diff
@@ -159,6 +159,7 @@ def install_node_prereqs():
     "edxval",                       # Because it was bork-installed somehow.
     "django-storages",
     "django-oauth2-provider",       # Because now it's called edx-django-oauth2-provider.
+    "edx-oauth2-provider",          # Because it moved from github to pypi
```

Comment:
Reminder to remove this line.  According to @clintonb, you don't need this change, right?